cmd/repl/repl.go

```diff
@@ -97,15 +115,15 @@ func main() {
 		}
 	}
 
-	for {
-		fmt.Print("sqlflow> ")
-		slct := readStmt()
-		fmt.Println("")
-
+	isTerminal := !flagPassed("execute", "e", "file", "f") && terminal.IsTerminal(syscall.Stdin)
```

Comment:
~~I am not sure if we need `terminal.IsTerminal()`. If the user doesn't type in `-e/-f`, we should start the program as a REPL.~~ Never mind, I see the code also supports reading from stdin.